sdk/src/main/java/com/hcaptcha/sdk/HCaptchaDialogFragment.java

```diff
@@ -169,6 +169,12 @@ public View onCreateView(@Nullable LayoutInflater inflater,
             prepareWebView(rootView, config);
 
             loadingContainer = rootView.findViewById(R.id.loadingContainer);
+            if (loadingContainer != null) {
+                final HCaptchaTheme theme = config.getTheme();
+                final int backgroundColor = (theme == HCaptchaTheme.DARK || theme == HCaptchaTheme.CONTRAST)
+                        ? Color.BLACK : Color.WHITE;
```

Comment:
future TODO: arguably we should extend the custom theming schema to cover the modal.

why CONTRAST? that is a light colored theme.

Reply:
Probably it should get its own key.